packages/@o3r/core/schematics/ng-add/index.ts

```diff
@@ -46,6 +46,7 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
     updateLinter(options, __dirname, o3rCoreVersion),
     updateAdditionalModules(options, __dirname),
     generateRenovateConfig(__dirname),
+    replaceWithNewPackages(['@otter/rules-engine', '@otter/rules-engine-core'], ['@o3r/rules-engine'], o3rCoreVersion),
```

Comment:
why this is not part of the `ng-add` of `@o3r/rules-engine`?

Reply:
because rules-engine is not added by default by o3r core.
and here I check if I have the old packages installed, I remove them and I install the new one.
I did not do the checks here because I need a helper from `o3r/dev-tools`, which is not a dep in `o3r/core`, so I delegate it to `o3r/schematics`.

Another solution would be to add a prompt like 'Do you want to add o3r/rules-engine ?' and if yes, install the o3r/rules-engine (and I can move this part to the ngAdd of rules-engine) . What do you think ?

Reply:
> Another solution would be to add a prompt like 'Do you want to add o3r/rules-engine ?' and if yes, install the o3r/rules-engine (and I can move this part to the ngAdd of rules-engine) . What do you think ?

I think it is a good idea :)

Reply:
ok. updated